app/src/main/java/org/dhis2/usescases/programEventDetail/ProgramEventDetailViewModel.kt

```diff
@@ -8,16 +17,24 @@ import androidx.lifecycle.viewModelScope
 import kotlinx.coroutines.flow.MutableSharedFlow
 import kotlinx.coroutines.flow.SharedFlow
 import kotlinx.coroutines.launch
+import org.dhis2.R
+import org.dhis2.commons.resources.ResourceManager
 import org.dhis2.commons.viewmodel.DispatcherProvider
 import org.dhis2.maps.layer.basemaps.BaseMapStyle
 import org.dhis2.maps.usecases.MapStyleConfiguration
+import org.dhis2.tracker.NavigationBarUIState
 import org.dhis2.usescases.programEventDetail.usecase.CreateEventUseCase
+import org.dhis2.utils.customviews.navigationbar.NavigationPage
+import org.dhis2.utils.customviews.navigationbar.NavigationPageConfigurator
+import org.hisp.dhis.mobile.ui.designsystem.component.navigationBar.NavigationBarItem
 
 class ProgramEventDetailViewModel(
```

Comment:
This is a good opportunity to add tests to this view model

Reply:
Yep, let me add a test for this change